Sources/MUXSDKStatsObjc/MUXSDKStats.h

```diff
@@ -321,12 +321,12 @@ FOUNDATION_EXPORT
 ///
 /// Use this method to signal that the player is now playing
 /// a new video. The player name provided must been passed
-/// as the name in a monitorPlayer:withPlayerName:andConfig:
-/// call. The config provided should match the specifications
-/// in the Mux docs at https://docs.mux.com and should include
```

Comment:
@jsanford8 was `and should include all desired keys, not just those keys that are specific to this video.` intended to mean values like custom environment key should be set on the new `MUXSDKCustomerData`, if they were set previously and are expected to continue being used? Want to make sure I'm interpreting the doc right.